strings.js

```diff
@@ -0,0 +1,263 @@
+// If the string length is less than 3, it should leave it unchanged.
+// For example:
+//
+// ```js
+// verbing('box')
+// // => 'boxing'
+// verbing('train')
+// // => 'training'
+// verbing('swimming')
+// // =>  'swimmingly'
+// verbing('go')
+// // =>  'go'
+// ```
+
+let singleStr = "box";
+if (singleStr.length < 3) {
+  console.log(singleStr);
+} else if (singleStr.length >= 3 && singleStr.slice(singleStr.length - 3) === "ing") {
+  console.log(singleStr + "ly");
+} else if (singleStr.length >= 3) {
+  console.log(singleStr + "ing");
+}
+
+let singleStr2 = "swimming";
+if (singleStr2.length < 3) {
+  console.log(singleStr2);
+} else if (singleStr2.length >= 3 && singleStr2.slice(singleStr2.length - 3) === "ing") {
+  console.log(singleStr2 + "ly");
+} else if (singleStr2.length >= 3) {
+  console.log(singleStr2 + "ing");
+}
+
+let singleStr3 = "go";
+if (singleStr3.length < 3) {
+  console.log(singleStr3);
+} else if (singleStr3.length >= 3 && singleStr3.slice(singleStr3.length - 3) === "ing") {
+  console.log(singleStr3 + "ly");
+} else if (singleStr3.length >= 3) {
+  console.log(singleStr3 + "ing");
+}
+
+
+// 7. Create a `withoutLast` code block that takes a single string variable,
+// and returns a copy of the string without the last word.
+
+let sentence = "You won't take me alive";
+let withoutLast = sentence.slice(0, sentence.length - 5);
+console.log(sentence + "\n" + withoutLast);
+
+// ## Bonus Questions
+//
+// 1. Write a code block that adds a "#" to a string if the string has an odd
+// number of characters, otherwise leave it as is. print/log the result
+//
+//     Examples:
+//
+//   ```js
+//   let name = 'taq'  // should log 'taq#'
+//   let name2 ='john' // should log 'john'
+//   ```
+
+let oddString = "phone";
+if (oddString.length % 2 === 1) {
+  console.log(oddString + "#");
+} else {
+  console.log(oddString);
+}
+
+
+// 2. Write a code block that prints/logs the 3rd to last characters of a string.
+// If the string is not at least 3 characters long then log an error message.
+// <details>
+//     <summary>
+//         Hint
+//     </summary>
+//     It should work for strings of any length.
+// </details>
+//
+// Examples, for variables:
+// ```js
+// let name1 = 'Taq' // Should log 'Taq'
+// let sentence = 'He is Taq'// Should log 'Taq'
+// ```
+
+let threat = "You need to back off before I eliminate you";
+if (threat.length < 3) {
+  console.log("#ERROR#");
+} else if (threat.length >= 3) {
+  console.log(threat.slice(threat.length - 3));
+}
+
+let threat2 = "Hi";
+if (threat2.length < 3) {
+  console.log("#ERROR#");
+} else if (threat2.length >= 3) {
+  console.log(threat2.slice(threat2.length - 3));
+}
+
+//
+// 3. Write a code block that prints/logs whether a string `x` starts with a
+// vowel.
+// <details>
+//   <summary>
+//     Extra
+//   </summary>
+//
+//   Make it [case insensitive](https://en.wikipedia.org/wiki/Case_sensitivity)
+//
+// </details>
+//
+
+let stringX = "all of you are fired!";
```

Comment:
Maybe using switch statement with string method "toLowerCase()" can shorten this code significantly.